cicd/Jenkins/Jenkinsfile

```diff
@@ -240,12 +243,15 @@ pipeline {
             }
             steps {
               script {
-                checkPackageVersion()
```

Comment:
Please check if this is still in use. If it's not, let's remove it.

Reply:
It is not. I've just removed it

Reply:
The function isVersionBump, takes into account the "VERSION_BUMP" stash, and it has been renamed. This will break the DevopsGCR because of this snippet of code: 
    
if (isVersionBump()) {
      //Set variable to trigger DevOps GCR Creation
      env.DEVOPS_GCR_PREPROD_BRANCH = 'master'
    }

Reply:
I see, I'm changing the new VERSION_BUMP_CHI into the good old VERSION_BUMP.
I'm keeping VERSION_BUMP_CHI_VUE, but that one is just to be used by the changes on this PR